photoslibraryapi/src/main/java/com/google/photos/library/v1/internal/InternalPhotosLibraryClient.java

```diff
@@ -1884,7 +1884,7 @@ public final MediaItem updateMediaItem(UpdateMediaItemRequest request) {
   }
 
   @Override
```

Comment:
Unfortunately the classes in the `internal` package are technically autogenerated before being published here, so it's a bit harder to update this class and we need to consider this carefully. 

@katmle - What do you think about this proposal? I can't quite think of a better way to address this either.

Reply:
I noticed that too but I can't think of any good reason to keep the close() "final" in the first place. I wonder if we could raise a CR to the gapic-generator maybe?

Reply:
@mixin82 Yes, after some more discussions the best way forward here is to see if we can submit this as a feature request to the gapic generator. 

(There are a few other changes on our end in the works that might also impact this - we are still using a slightly different generator and need to switch over first.) I'll file a feature request in the next few days once we have confirmed some additional details on our end.

In the meantime we might be able to submit this as-is and spin up a new release with this change. Stay tuned, we'll do a little more testing to make sure there aren't any other unintended side effects.